build/dsc.rake

```diff
@@ -110,8 +110,8 @@ eod
     task :build, [:module_path] do |t, args|
       module_path = args[:module_path] || default_dsc_module_path
       m = Dsc::Manager.new
-      wait_for_resources = Dir["#{module_path}/**/*WaitFor*"]
-      fail "WaitFor* resources found - aborting type building!\n\n#{wait_for_resources}\n" if !wait_for_resources.empty?
+      wait_for_resources = Dir["#{module_path}/**/MSFT_WaitFor*"]
+      fail "MSFT_WaitFor* resources found - aborting type building! Please remove any MSFT_WaitFor* DSC Resources and run the build again.\n\n#{wait_for_resources}\n" if !wait_for_resources.empty?
```

Comment:
Since we're dumping the list of resources, can probably reword:

`Please remove any`

To

`Please remove the following`